frontend/packages/dev-console/src/utils/shared-submit-utils.ts

```diff
@@ -43,8 +43,9 @@ export const createService = (
 
   let ports = imagePorts;
   const buildStrategy = formData.build?.strategy;
-  if (buildStrategy === 'Docker') {
-    const port = { containerPort: _.get(formData, 'docker.containerPort'), protocol: 'TCP' };
+  if (buildStrategy === 'Docker' && unknownTargetPort) {
+    const containerPort = _.toInteger(unknownTargetPort) || defaultUnknownPort;
```

Comment:
I seems that you dont need to add `defaultUnknownPort` here since there is already a condition below which checks if the ports are empty then use `defaultUnknownPort`.

Reply:
Removed `defaultUnknownPort`. Tested without providing `containerPort` and `defaultUnknownPort` is used if no ports are stated.